readthedocs/core/mixins.py

```diff
@@ -39,7 +41,11 @@ class HideProtectedLevelMixin:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        privacy_level = list(PRIVACY_CHOICES)
+        privacy_level.remove((PROTECTED, _('Protected')))
```

Comment:
Are we ready to just remove this from the list itself?

Reply:
It's already removed, yeah.